src/main/java/gov/cabinetoffice/gap/adminbackend/repositories/GrantAdminRepository.java

```diff
@@ -4,13 +4,16 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
 import java.util.Optional;
 
 @Repository
 public interface GrantAdminRepository extends JpaRepository<GrantAdmin, Integer> {
 
     Optional<GrantAdmin> findByGapUserUserSub(String sub);
 
+    Optional<List<GrantAdmin>> findAllByGapUser_IdIn(List<Integer> userIds);
```

Comment:
Dont need to wrap this in optional as an empty list would simply be returned in that scenario
```suggestion
   List<GrantAdmin> findAllByGapUserIdIn(List<Integer> userIds);
```